core/state_processor.go

```diff
@@ -344,37 +344,79 @@ func (p *StateProcessor) Process(block *types.Block, etxSet *types.EtxSet) (type
 		i++
 	}
 
-	qiTransactions := block.QiTransactions()
+	coinbaseTx := block.Transactions()[0]
```

Comment:
check if there is atleast one transaction in the block

Reply:
Fixed